### PR TITLE
#284 Add persuasive Massachusetts Appellate Court cases

### PR DESCRIPTION
**Pull request by flooie**
#284 Add persuasive Massachusetts Appellate Court cases to Juriscraper.

Massachusetts has an excellent website hosting 15,227 (as of now) persuasive decisions from the Appellate court.   
This scraper adds support for scraping for this database.

### CODE REVIEW

File: juriscraper/opinions/united_states/state/massappct_u.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            './/div[contains(text(),"Release Date")]/following-sibling::div[1]'
+        )
+        return [
+            convert_date_string(case_date.text_content().strip())
+            for case_date in self.html.xpath(xp)
+        ]
+
+    def _get_precedential_statuses(self):
+        xp = './/div[contains(text(),"Case Name")]/following-sibling::div[1]'
+        return ["Persuasive" for case_name in self.html.xpath(xp)]
```
Comment (mlissner) [inline juriscraper/opinions/united_states/state/massappct_u.py:62]: OK, this is what we agreed to change. We decided non-precedential is better here, right?

File: tests/examples/opinions/united_states/massappct_u_example.compare.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
+[
+  {
+    "case_dates": "2020-12-04",
+    "case_names": "ROBERT FERRIER & others v. COMCAST CORPORATION & others.",
+    "download_urls": "/Disposition/DownloadDisposition/16253?dispositionId=16267",
```
Comment (mlissner) [inline tests/examples/opinions/united_states/massappct_u_example.compare.json:5]: Aren't download URLs supposed to be full URLs? If I'm right about that, maybe we should add a test for this across all the sample files as well as fixing this.

**Review by mlissner — CHANGES_REQUESTED** [review]
Just a couple little things, I think. Thanks for the initiative on these.

**Comment by mlissner** [comment]
CLA Assistant reminded me this PR is floating around, @flooie.

**Comment by flooie** [comment]
Seriously I have so many wanted courts.  Mostly in the veteran world

File: tests/examples/opinions/united_states/massappct_u_example.compare.json (hunk 2 above)
Comment (flooie) [reply tests/examples/opinions/united_states/massappct_u_example.compare.json:5]: yes.

File: juriscraper/opinions/united_states/state/massappct_u.py (hunk 1 above)
Comment (flooie) [reply juriscraper/opinions/united_states/state/massappct_u.py]: yes - its not set to unpublished.

File: tests/examples/opinions/united_states/massappct_u_example.compare.json (hunk 2 above)
Comment (flooie) [reply tests/examples/opinions/united_states/massappct_u_example.compare.json]: this is fixed.

**Comment by flooie** [comment]
@mlissner -- its amazing how much changes in a year.

This has been updated and fixed.  tests fixed, now streamlined with opinionsitelinear and I think should be good to go.

**Pull request merged**